src/main/java/seedu/address/logic/commands/DeleteTagCommand.java

```diff
@@ -19,9 +19,8 @@
 public class DeleteTagCommand extends Command {
     public static final String COMMAND_WORD = "tagdel";
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": deletes tags linked to the task.\n"
-            + "Format: " + COMMAND_WORD + " " + "[" + PREFIX_TAG + "TAG]\n"
-            + "Example: " + COMMAND_WORD + " " + "t/" + "priority\n"
-            + COMMAND_WORD + " " + PREFIX_TAG + "priority deadline";
+            + "Parameters: " + COMMAND_WORD + " " + "INDEX " + PREFIX_TAG + "KEYWORD [SECOND_KEYWORD]\n"
+            + "Example: " + COMMAND_WORD + " " + "t/" + "priority\n";
```

Comment:
example should be with a t infront of the command word

Reply:
yup i just changed all of the message usages

Reply:
wait i mean in ur example

Reply:
wait sorry I have just noticed it. I have included it in.